mojave/machinery/terminals.dm

```diff
@@ -409,3 +409,8 @@
 /obj/machinery/ms13/terminal/wall/rust
 	icon_state = "wallterminal_rusted"
 	base_icon_state = "wallterminal_rusted"
+
+/obj/machinery/ms13/terminal/wall/classic
+	icon_state = "terminal_classic"
+	base_icon_state = "terminal_classic"
+	screen_icon = "terminal_classic_screen"
```

Comment:
Kinda didn't want to add these until the system was made a bit better so that it can be made a bit more unique. The background will currently be green with the robco branding and model number, despite the obvious physical differences. Not lore friendly bro!

Reply:
ok terminal colours are modular now and all wall terminals dont have to be flippable

Reply:
Still gotta deal with the ROBCO loredrop and version numbers and whatever. But okay.